docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -489,6 +477,8 @@ Enables YugabyteDB to [automatically split tablets](../../../architecture/docdb-
 
 Default: `true`
 
+Note that this value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+
```

Comment:
Maybe worth wrapping into Important block. Is there any new style to not use note block as for 2.14?

Reply:
Not really , I can include it back into a note box for getting it more attention. Thanks.